bigbluebutton-html5/imports/ui/components/chat/chat-graphql/chat-message-list/page/chat-message/message-reactions/component.tsx

```
@@ -57,57 +57,69 @@ const ChatMessageReactions: React.FC<ChatMessageReactionsProps> = (props) => {
 
   if (reactions.length === 0) return null;
 
-  const reactionItems: Record<
-    string,
-    ReactionItem
-  > = {};
+  const reactionItems = reactions.reduce((previousValue, reaction) => {
+    const newValue = { ...previousValue };
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)